source/cloud/cloud-administration/sso-office.md

```diff
@@ -3,36 +3,75 @@ ___
+Once the App Registration has been created, you can configure it further. The standard Azure AD documentation is [here](https://docs.microsoft.com/en-gb/azure/active-directory/develop/quickstart-register-app) for reference.
+
+### Step 2: Generating a new client secret in Azure Portal
 
-4. Select **Certificates and Secrets** from the menu, and click the button to generate a **New Client secret**. Provide a description and define the expiry for the token. 
+1. In the Azure Portal, select **Certificates and Secrets** from the menu, then select the button to generate a **New Client secret**. 
+
+2. Provide a description, define the expiry for the token, then select **Add**.
 
    ![](../../../source/images/AzureApp_Client_Secret_Expiry.png)
 
-Click _Add_ and you will be provided with the _client secret value_, copy this and save it for use in the Mattermost configuration as the **Application Secret Password**.
+You are provided with the _client secret value_. Copy and paste it to a temporary location. You will enter this value as a **Client Secret** in the Mattermost System Console.
+
+### Step 3: Copying values for Mattermost configuration
 
-5. Select **Overview** from the menu and copy the _Application (client) ID_ and the _Directory (tenant) ID_, for use in the Mattermost configuration as the **Application ID** and as part of the **Auth Endpoint** and **Token Endpoint** URL.
+In Azure Portal, select **Overview** from the menu, then copy and paste both the _Application (client) ID_ and the _Directory (tenant) ID_ to a temporary location. You will enter these values as an **Application ID** and as part of an **Auth Endpoint** and **Token Endpoint** URL in the Mattermost System Console.
 
    ![](../../../source/images/AzureApp_App_Directory_IDsv2.png)
 
-6. Log in to Mattermost and then go to the **System Console > OAuth 2.0 > Select OAuth 2.0 service provider**, choose **Office 365** as the service provider. Enter the _client secret value_ you copied in Step 4 as the **Application Secret Password**. Paste the _Application (client) ID_ you saved in Step 5 into the **Application ID** field.
+### Step 4: Configuring Mattermost for Office 365 SSO
+
+1. Log in to Mattermost, then go to the **System Console > Authentication > OpenID Connect**.
+
+2. Select **Office 365** as the service provider.
+
+3. Paste in _client secret value_ as the **Client Secret**.
+
+4. Paste in the _Application (client) ID_ as the **Client ID**
+
+5. Paste in the _Directory (tenant) ID_ as the **Directory (tenant) ID**.
+
+6. Open the Mattermost `config.json` file.
+
+7. In the `Office365Settings` section of `config.json` file, update the following default values:
+
+   - ``AuthEndpoint: https://login.microsoftonline.com/common/oauth2/v2.0/authorize``
```

Comment:
@sbishel - Anything here that needs to change with regards to supporting OpenID Connect over OAuth 2?